https://bugs.gentoo.org/show_bug.cgi?id=369883

Remove "5 years old sources" warning that causes failures when building
formats...

by Duncan Exon Smith


--- texmf-dist/tex/latex/base/latex.ltx	2011-06-10 16:29:49.763911520 +0200
+++ texmf-dist/tex/latex/base/latex.ltx	2011-06-10 16:30:13.854910835 +0200
@@ -522,30 +522,6 @@ 
 %%% From File: ltvers.dtx
 \def\fmtname{LaTeX2e}
 \edef\fmtversion{2005/12/01}
-\iftrue
-\def\reserved@a#1/#2/#3\@nil{%
-  \count@\year
-  \advance\count@-#1\relax
-  \multiply\count@ by 12\relax
-  \advance\count@\month
-  \advance\count@-#2\relax}
-\expandafter\reserved@a\fmtversion\@nil
-\ifnum\count@>65
-  \typeout{^^J%
-!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!^^J%
-!  You are attempting to make a LaTeX format from a source file^^J%
-!  That is more than five years old.^^J%
-!^^J%
-!  If you enter <return> to scroll past this message then the format^^J%
-!  will be built, but please consider obtaining newer source files^^J%
-!  before continuing to build LaTeX.^^J%
-!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!^^J%
-}
-   \errhelp{To avoid this error message, obtain new LaTeX sources.}
-   \errmessage{LaTeX source files more than 5 years old!}
-\fi
-\let\reserved@a\relax
-\fi
 \everyjob{\typeout{\fmtname
                                      \space<\fmtversion>}}
 \immediate\write16{\fmtname
